sklearn/ensemble/_gb.py

```diff
@@ -2032,6 +2032,11 @@ class GradientBoostingRegressor(RegressorMixin, BaseGradientBoosting):
     array([-61...])
     >>> reg.score(X_test, y_test)
     0.4...
+
+    For a detailed example of utilizing
+    :class:`~sklearn.ensemble.GradientBooestingRegressor`
```

Comment:
I guess this `e` in `GradientBooestingRegressor` is a typo...

Reply:
Done.